### PR TITLE
Add laceyhenschel.com.json

### PR DESCRIPTION
**Pull request by williln**

### CODE REVIEW

File: blogs/laceyhenschel.com.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "photo_url": "https://avatars.githubusercontent.com/u/2286304?v=4",
+  "description": "Occasional posts about Python and Django",
+  "language": "en",
+  "authors": [ 
+    {
+      "name": "Lacey Williams Henschel",
+      "social_media": [{
+         "github": "williln",
+         "website": "https://www.laceyhenschel.com",
+         "linkedin": "laceynwilliams",
```
Comment (cosimameyer) [inline blogs/laceyhenschel.com.json:14]: The trailing comma should be removed here - otherwise it will throw errors when using the data :)

**Comment by cosimameyer** [comment]
Thanks so much for contributing, @williln! 🥳 
Just two small changes/requests: 

- Would you be interested in adding your RSS feed? This way, the [PyLadies' bot](https://botsin.space/@pyladies_bot) on Mastodon can promote your content (I looked it up and it should be `"rss_feed": "http://feeds.feedburner.com/LaceyWilliamsHenschel"`) 
- I added a comment where a trailing comma should be removed :) 

If you have any questions, please let me know - thank you again for contributing to this project and making publicly accessible content!

**Comment by cosimameyer** [comment]
@williln I hope you're doing well ☺️ I'm just checking in to see if you have any questions about the PR - let me know if you need any input from my side 🙌

**Comment by williln** [comment]
@cosimameyer Sorry for the delay! I believe I fixed the things you mentioned and added my RSS feed (thanks for looking it up for me), and rebased my changed. Let me know what else you need and I will try not to wait several months to fix it again 😄

**Comment by cosimameyer** [comment]
Thank you so much for contributing - it all looks fantastic! 🥳

**Review by cosimameyer — APPROVED** [review]

**Pull request merged**